src/modules/core/components/stepper/stepper.interface.ts

```diff
@@ -0,0 +1,20 @@
+import type { TypographyProps } from '@/modules/core/components/typogrpahy/typography.interface';
+
+export type StepperStep = {
+  value: string;
+  text: string;
+};
+
+export type StepperProps = {
+  value: string;
+  steps: StepperStep[];
+  allowClickOnPassedSteps?: boolean;
```

Comment:
I don't think we need this prop. Better to handle go back with button as it's not obvious for the user that this steps might be clickable.

Reply:
@IhorBalan removed